prow/config/jobs.go

```diff
@@ -150,9 +150,9 @@ func (c *Config) SetPresubmits(jobs map[string][]Presubmit) error {
 	return nil
 }
 
-func (c *Config) AllJobNames() []string {
+func (c *Config) AllPresubmit() []string {
```

Comment:
Almost, but it should be `AllPresubmits` since it returns a list of `Presubmit`.